### PR TITLE
[MODORDERS-1195] - Fix test related to restriction for last piece deletion

### PR DESCRIPTION
**Pull request by azizbekxm**
## Purpose
https://folio-org.atlassian.net/browse/MODORDERS-1195

## Approach
- Increased quantity or changed checkinIteam to true
- Improvements

### CODE REVIEW

**Comment by azizbekxm** [comment]
karate test result
image

Orders test
image
image
image

Consortia
image

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/piece-deletion-restriction.feature [hunk 1]
```diff
@@ -8,24 +8,23 @@ Feature: Piece deletion restrictions from order and order line
...
+    * def fundId = call uuid
+    * def budgetId = call uuid
 
+    # Prepare finances
+    * configure headers = headersAdmin
+    * call createFund { 'id': '#(fundId)' }
+    * call createBudget { 'id': '#(budgetId)', 'allocated': 10000, 'fundId': '#(fundId)' }
+    * configure headers = headersUser
 
   Scenario: Avoid deltion of piece when order has status 'Closed' and poLine cost quantity '1' and ReceivingWorkflow 'Syncronized'
```
Comment (BKadirkhodjaev) [inline acquisitions/src/main/resources/thunderjet/mod-orders/features/piece-deletion-restriction.feature:23]: A small typo: "Avoid deletion..." & can add @Positive/@Negative indicators as usual

**Review by BKadirkhodjaev — APPROVED** [review]

File: acquisitions/src/main/resources/thunderjet/cross-modules/features/pay-invoice-and-delete-piece.feature [hunk 2]
```diff
@@ -35,31 +27,31 @@ Feature: Pay an invoice and delete a piece
...
 
-
-  Scenario: Receive the piece
+    # 4. Receive the piece
     Given path 'orders/pieces'
     And param query = 'poLineId==' + poLineId
     When method GET
     Then status 200
-    And match $.totalRecords == 1
+    And match $.totalRecords == 2
```
Comment (BKadirkhodjaev) [inline acquisitions/src/main/resources/thunderjet/cross-modules/features/pay-invoice-and-delete-piece.feature:54]: I guess we change to 2 to allow 1 for deletion and 1 to remain

**Review by SerhiiNosko — APPROVED** [review]

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/piece-deletion-restriction.feature (hunk 1 above)
Comment (azizbekxm) [reply acquisitions/src/main/resources/thunderjet/mod-orders/features/piece-deletion-restriction.feature:23]: Thanks, fixed

Comment (azizbekxm) [reply acquisitions/src/main/resources/thunderjet/mod-orders/features/piece-deletion-restriction.feature]: As for Annotation, both scenarios will check for sucess and exception case, this is why didn't add

File: acquisitions/src/main/resources/thunderjet/cross-modules/features/pay-invoice-and-delete-piece.feature (hunk 2 above)
Comment (azizbekxm) [reply acquisitions/src/main/resources/thunderjet/cross-modules/features/pay-invoice-and-delete-piece.feature:54]: There is two pieces, currently getting total 2 and deleting one. didn't introduce extra case, just fixed

**Pull request merged**